kafka-env/kafka-connect/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM confluentinc/cp-kafka-connect
+
+ENV CONNECT_PLAGIN_PATH: "usr/share/java,/usr/share/confluent-hub-components"
+
+RUN confluent-hub install --no-prompt debezium/debezium-connector-postgresql:2.2.1
```

Comment:
minor: In theory, also could be simplified into a single docker-compose service with 
```
command: 
RUN confluent-hub install --no-prompt debezium/debezium-connector-postgresql:2.2.1
```
...

Reply:
In theory - yes.
In practise - not really.
Removed `ENV`.

Reply:
`
2023-06-29 13:33:40 Running in a "--no-prompt" mode 
2023-06-29 13:33:47 Implicit acceptance of the license below:  
2023-06-29 13:33:47 Apache 2.0 
2023-06-29 13:33:47 https://github.com/debezium/debezium/blob/master/LICENSE.txt 
2023-06-29 13:33:47 Implicit confirmation of the question: You are about to install 'debezium-connector-postgresql' from Debezium Community, as published on Confluent Hub. 
2023-06-29 13:33:47 Downloading component Debezium PostgreSQL CDC Connector 2.2.1, provided by Debezium Community from Confluent Hub and installing into /usr/share/confluent-hub-components 
2023-06-29 13:33:49 Adding installation directory to plugin path in the following files: 
2023-06-29 13:33:49   /etc/kafka/connect-distributed.properties 
2023-06-29 13:33:50   /etc/kafka/connect-standalone.properties 
2023-06-29 13:33:50   /etc/schema-registry/connect-avro-distributed.properties 
2023-06-29 13:33:50   /etc/schema-registry/connect-avro-standalone.properties 
2023-06-29 13:33:50  
2023-06-29 13:33:50 Completed
`
The container simply stops after executing the command.